modules/@apostrophecms/attachment/index.js

```diff
@@ -987,7 +1059,7 @@ module.exports = {
               await method(path);
             } catch (e) {
               // afterSave is not a good place for fatal errors
-              self.apos.util.warn('Unable to set permissions on ' + path + ', possibly it does not exist');
+              self.apos.util.warn(`Unable to ${action} ${path}, most likely already done`);
```

Comment:
This is a console warning, we don't currently have plans to translate those in 3.x stable. I think it would be a lot to add to our plate and we could revisit later.